docs/syncset.md

```diff
@@ -0,0 +1,94 @@
+# SyncSet
+
+## Overview
+
+`SyncSet` and `SelectorSyncSet` objects facilitate resource management (create, update, delete, patch) in hive-managed clusters.
+
+## SyncSet Object Definition
+
+A `SyncSet` manages objects in clusters referenced by `ClusterDeployment` name that share a namespace with the `SyncSet`. `SyncSets` may contain a list of resource object definitions to create and a list of patches which are applied to existing objects.
+
+Individual resources and patches will be reapplied if 2 hours have passed since their last reconcilation or if their contents were updated within the `SyncSet`.
```

Comment:
If this applies to both `SyncSets` and `SelectorSyncSets`, I'd say it in the intro.